client/components/common/TestPlanResultsTable/index.jsx

```diff
@@ -56,11 +63,19 @@ const TestPlanResultsTable = ({ test, testResult, tableClassName = '' }) => {
 
                 return (
                     <React.Fragment key={scenarioResult.id}>
-                        <h3>
-                            {commandsString}&nbsp;Results:&nbsp;
-                            {passedAssertions.length} passed,&nbsp;
-                            {failedAssertions.length} failed
-                        </h3>
+                        {isReportsPage ? (
```

Comment:
Instead of changing the behavior based on an `isReportsPage` prop, which is difficult to understand without delving in here, maybe we can add a `headingLevel={3}` prop. It would work like this ...

```js
const Header = `h${headingLevel}`

return (
  // ...
  <Header>
    Header content
  </Header>
)
```

Reply:
Thanks for that, been seeing this sprinkled around! We could make general component for this pattern now. Also updated in https://github.com/w3c/aria-at-app/pull/845/commits/9eb58751f7aabdd88fe8aaf53d1afe21d67d7b7f.